### PR TITLE
add project file to week 8

### PR DESCRIPTION
**Pull request by jsms90**
+ Add project file to week 8
+ Fix links to JWT Workshop

Relates #34

### CODE REVIEW

File: coursebook/week-8/project.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+> **So that** I benefit from logging in with Github OAuth, and don't have to do any profile setup on your site.
+
+Acceptance criteria:
+
++ [ ] I can see my username & profile picture on each page that I visit
+
+#### Stretch goals - choose _one_ (ONLY IF YOU HAVE TIME)
+
+**As a** paranoid member of the current cohort, who believes in "what happens at FAC stays at FAC"
+> **I want** other's viewing rights to my posts to be restricted
```
Comment (emilyb7) [inline coursebook/week-8/project.md]: others'

but maybe we want to be clear about who 'others' are. Maybe 'users from outside the Founders & Coders organisation'?

File: coursebook/week-8/project.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+> **So that** I feel free to be wildly opinionated with my crazy blogging.
+
+Acceptance criteria:
+
++ [ ] Only logged-in members of Founders and Coders can see any of the content on this site
++ [ ] Anyone who is logged in, but not part of the current cohort, should still see a list of blogs. But they should see nothing that I (the paranoid user) has posted.
```
Comment (emilyb7) [inline coursebook/week-8/project.md]: But they should see nothing that I (the paranoid user) **have** posted.

File: coursebook/week-8/project.md [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
++ For the purposes of your learning, implementing OAuth is your top priority!
++ It is your choice whether you would like to perfect every detail of the 2 mandatory user stories, or to move on to a third user story. Either way, your choice should be explained in your readme.
+
+### How are both pairs going to work on the same project on Github?
+
+Normally, if you were working off someone's existing repo, you would fork it. But forks work under the assumption that they will, at some point, get merged into the original. So you cannot raise issues on a forked repo. That's no good for our purposes in week 8.
+
+Instead:
+1. Create a new, blank repo. Initialise it _without_ a `README.md` or a `.gitignore`
+2. Clone your week 7 project on Github. You will need perform `git clone` from inside a different local directory than the one that contains your first version of the week 7 project.
```
Comment (emilyb7) [inline coursebook/week-8/project.md]: You will need **to** perform `git clone`...

Is `perform` even the right term? What about `run`?

File: coursebook/week-8/project.md (hunk 2 above)
Comment (jsms90) [inline coursebook/week-8/project.md]: But I **has** posted it @emilyb7 !! I has! I promise :stuck_out_tongue_winking_eye:

**Comment by jsms90** [comment]
Removed the reference to a particular percentage in terms of code coverage - unsure as to whether I would be able to answer how to test all of this (testing in FAC9 wasn't strong :disappointed: )

**Comment by bradreeder** [comment]
This needs to be used tomorrow so I'm merging without a second review.

**Pull request merged**